nixos/doc/manual/from_md/release-notes/rl-2111.section.xml

```diff
@@ -555,6 +555,14 @@
           6.0.0 to 9.0.0
         </para>
       </listitem>
+      <listitem>
+        <para>
+          <literal>tt-rss</literal> was upgraded to the commit on
+          2021-06-21, which has breaking changes, see
+          <link xlink:href="https://community.tt-rss.org/t/rip-config-php-hello-classes-config-php/4337">this
```

Comment:
It would be nice to be more specific about what adaptations must be done. I'm thinking about something like "If you use `services.ttrss.extraConfig` you should migrate to `putenv` style configuration. See this discourse post for details."